modules/ui/bin/jenkins-build-deploy.sh

```diff
@@ -36,10 +38,10 @@ exitOnError() {
 }
 
 fromPom() {
-  mvn -f ./modules/$1/pom.xml -P $2 help:evaluate -Dexpression=$3 | sed -n -e '/^\[.*\]/ !{ p; }'
+  mvn -f ./pom.xml help:evaluate -Dexpression=$1 | sed -n -e '/^\[.*\]/ !{ p; }' | tail -n 1
 }
 
-version=`fromPom main ${profile} project.version`
+version=`fromPom project.version`
```

Comment:
@bargenilesh Why was `profile` removed for getting the version? I understand that we removed the `main` attribute as we are probably getting the version information from the root pom instead of the `modules/main/pom` now but maven profile would still be helpful, right?